src/Instrumentation/Laravel/README.md

```diff
@@ -49,3 +49,14 @@ From Laravel subdirectory:
 $ composer install
 $ ./vendor/bin/phpunit tests
 ```
+
+## Configuration
+
+Parts of this auto-instrumentation library can be configured, more options are available throught the
+[General SDK Configuration](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#general-sdk-configuration):
+
+| Name                                | Default value | Values                  | Example | Description                                                                     |
+|-------------------------------------|---------------|-------------------------|---------|---------------------------------------------------------------------------------|
+| OTEL_PHP_DISABLED_INSTRUMENTATIONS  | []            | Instrumentation name(s) | laravel | Disable one or more installed auto-instrumentations, names are comma seperated. |
+
+Configurations can be provided as environment variables, or via `php.ini` (or a file included by `php.ini`)
```

Comment:
I'm not sure that we can set this via `php.ini` (at least I'm not aware of that)

Reply:
Thats what the main docs says about this feature

Reply:
ok, you are right